uwblueprint/childrens-aid-society
# Implement Dummy Court Information Screen

## Notion ticket link

[Implement Dummy Court Information Screen](https://www.notion.so/uwblueprintexecs/Implement-dummy-court-information-screen-752b3d08e88c446d88cb375e34ff8495)



## Implementation description
* Make a new component (components/Intake/CourtInformationForm.tsx) with the form fields from the Figma link. Store the values for the fields that are entered in the Intake page.
* [Figma link](https://www.figma.com/file/1obAgfYZ639hpKXV1ZP7hJ/%F0%9F%8C%AC-Intake-Flow?node-id=519%3A11840)
* Note: The file upload put a temporary basic file upload one. Will need to be updated



## Steps to test
1. startup docker containers
2. go to localhost:3000/intake, visit step 2 and you should see the information screen



## What should reviewers focus on?
* The localhost:3000/intake step 2 stage
* Passed on the props properly?


## Checklist
